content/en/docs/overview.md

```diff
@@ -1,29 +1,29 @@
 ---
 title: "Project overview"
-description: "An overview of the Notary project"
+description: "An overview of The Notary Project"
 type: docs
 weight: 1
 ---
 
 {{% alert title="Important" color="info" %}}
-This article as well as the rest of this documentation describes the latest status of Notary Project and refers to it as The Notary Project. For documentation of older versions and implementations, please refer to the [previous Notary documentation](https://github.com/notaryproject/notary/tree/master/docs).
+This article as well as the rest of this documentation describes the latest status of The Notary Project and refers to it as The Notary Project. For documentation of older versions and implementations, please refer to the [previous documentation](https://github.com/notaryproject/notary/tree/master/docs).
 {{% /alert %}}
 
 ## Project Status
 
-The Notary project is in early development and design documents should not be considered final.
+The Notary Project is in early development and design documents should not be considered final.
```

Comment:
I can understand T is capital where it is at the beginning of the sentence.

Reply:
Leaving as is